store/sqlstore/channel_member_history_store.go

```diff
@@ -109,47 +115,42 @@ func (s SqlChannelMemberHistoryStore) hasDataAtOrBefore(time int64) (bool, error
 }
 
 func (s SqlChannelMemberHistoryStore) getFromChannelMemberHistoryTable(startTime int64, endTime int64, channelId string) ([]*model.ChannelMemberHistoryResult, error) {
-	query := `
-			SELECT
-				cmh.*,
-				u.Email,
-				u.Username,
-				Bots.UserId IS NOT NULL AS IsBot,
-				u.DeleteAt AS UserDeleteAt
-			FROM ChannelMemberHistory cmh
-			INNER JOIN Users u ON cmh.UserId = u.Id
-			LEFT JOIN Bots ON Bots.UserId = u.Id
-			WHERE cmh.ChannelId = :ChannelId
-			AND cmh.JoinTime <= :EndTime
-			AND (cmh.LeaveTime IS NULL OR cmh.LeaveTime >= :StartTime)
-			ORDER BY cmh.JoinTime ASC`
-
-	params := map[string]interface{}{"ChannelId": channelId, "StartTime": startTime, "EndTime": endTime}
+	query, args, err := s.getQueryBuilder().
+		Select("cmh.*, u.Email, u.Username, Bots.UserId IS NOT NULL AS IsBot, u.DeleteAt AS UserDeleteAt").
+		From("ChannelMemberHistory cmh").
+		Join("Users u ON cmh.UserId = u.Id").
+		LeftJoin("Bots ON Bots.UserId = u.Id").
+		Where(sq.Eq{"cmh.ChannelId": channelId}).
```

Comment:
Are you able to use a SQ.And here instead of 3 different where statements similar to how you've done it later on?